libbeat/outputs/elasticsearch/docs/elasticsearch.asciidoc

```diff
@@ -37,13 +37,14 @@ output.elasticsearch:
   password: "{pwd}"
 ------------------------------------------------------------------------------
 
-To use an API key to connect to {es}, use `api_key`.
+To use an API key to connect to {es}, use `api_key`. The value must be the ID of
+the API key and the API key joined by a colon.
```

Comment:
This info would be good down on line 137 with the `api_key`.

Reply:
Done in 42d288616.